robosat/tools/export.py

```diff
@@ -32,7 +32,7 @@ def map_location(storage, _):
 
     chkpt = torch.load(args.checkpoint, map_location=map_location)
     net = torch.nn.DataParallel(net)
-    net.load_state_dict(chkpt)
+    net.load_state_dict(chkpt['state_dict'])
```

Comment:
Would be great if you could format all your changes with black. The travis config shows the command line argument for it; thanks!

Reply:
Fixed !